boards/arm/circuitdojo_feather_nrf9160/Kconfig.defconfig

```diff
@@ -9,6 +9,21 @@ if BOARD_CIRCUITDOJO_FEATHER_NRF9160 || BOARD_CIRCUITDOJO_FEATHER_NRF9160_NS
 config BOARD
 	default "circuitdojo_feather_nrf9160"
 
+# By default, if we build for a Non-Secure version of the board,
+# enable building with TF-M as the Secure Execution Environment.
+config BUILD_WITH_TFM
+	default y if BOARD_CIRCUITDOJO_FEATHER_NRF9160_NS
```

Comment:
Cannot this go upstream?

Reply:
It cannot go upstream, it does not compile. We only support this in downstream.